content/developers/metrics/count.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Count
+kind: documentation
+further_reading:
+- link: "developers/metrics"
+  tag: "Documentation"
+  text: Learn more about Metrics
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: Official and Community-contributed API and DogStatsD client libraries
+---
+
+## Overview
+
+Counters are used to count things.
+
+## Submission
+
+### Agent check
+
+{{% table responsive="true" %}}
+|Method | Overview |
+|:---|:---|
```

Comment:
Tables were used in the former metrics page in order to augment compacity. Now that each metric types have its own dedicated documentation page, maybe switching back to a "full-text" doc would enhance readability.